src/test/resources/cucumber/features/portalAdmin/DARTS_admin.feature

```diff
@@ -199,3 +214,174 @@ Feature: Admin
+    When I check the checkbox in the same row as "Testgroupone" "Transcriber"
+    And I press the "Assign groups" button
+    Then I see "Assigned 1 group" on the page
+
+  @DMP-2323
+  Scenario: Deactivate user with transcript
+
+    #Create transcript request and assign to user
+
+    Given I am logged on to DARTS as an REQUESTER user
+    And I click on the "Search" link
+    And I set "Case ID" to "H{{seq}}001"
+    And I press the "Search" button
+    And I click on "H{{seq}}001" in the same row as "Harrow Crown Court"
+    And I click on the "{{displaydate}}" link
+    And I click on the "Transcripts" link
+    And I press the "Request a new transcript" button
+    Then I see "Audio list" on the page
+
+    When I select "Specified Times" from the "Request Type" dropdown
+    And I select "Overnight" from the "Urgency" dropdown
+    And I press the "Continue" button
+    Then I see "Events, audio and specific times requests" on the page
+
+    When I set the time fields below "Start time" to "10:30:00"
+    And I set the time fields below "End time" to "10:31:00"
+    And I press the "Continue" button
+    Then I see "Check and confirm your transcript request" on the page
+    And I see "H{{seq}}001" in the same row as "Case ID"
+    And I see "Harrow Crown Court" in the same row as "Courthouse"
+    And I see "DefH {{seq}}-46" in the same row as "Defendant(s)"
+    And I see "{{displaydate}}" in the same row as "Hearing date"
+    And I see "Specified Times" in the same row as "Request type"
+    And I see "Overnight" in the same row as "Urgency"
+    And I see "Provide any further instructions or comments for the transcriber." on the page
+
+    When I set "Comments to the Transcriber (optional)" to "This transcript request is for user to be deactivated"
+    And I check the "I confirm I have received authorisation from the judge." checkbox
+    And I press the "Submit request" button
+    Then I see "Transcript request submitted" on the page
+
+    When I click on the "Return to hearing date" link
+    Then I see "Transcripts for this hearing" on the page
+    And I see "Specified Times" in the same row as "Awaiting Authorisation"
+
+    When I Sign out
+    And I see "Sign in to the DARTS Portal" on the page
+    And I am logged on to DARTS as an APPROVER user
+    And I click on the "Your transcripts" link
+    And I click on the "Transcript requests to review" link
+    And I click on "View" in the same row as "H{{seq}}001"
+    And I see "This transcript request is for user to be deactivated" in the same row as "Instructions"
+    And I select the "Yes" radio button
+    And I press the "Submit" button
+    And I click on the "Transcript requests to review" link
+    Then I see "Requests to approve or reject" on the page
+    And I do not see "H{{seq}}001" on the page
+
+    When I Sign out
+    And I see "Sign in to the DARTS Portal" on the page
+    #And I am logged on to DARTS as TESTUSERTHREE user
```

Comment:
Do we need this line to sign in after signing out?

Reply:
That is a placeholder for if/when we get the auto to log in to a brand new transcriber user (if it is possible). Currently we only have established users (requester/approver/transcriber). I left the regression tag out of this scenario so it won't run until it is ready.